Docs: Simplify getting started docs (#7618)
github/codeql: docs/codeql/codeql-cli/getting-started-with-the-codeql-cli.rst

   ":ref:`Setting up CodeQL in Visual Studio Code <setting-up-codeql-in-visual-studio-code>`."

4. Verify your CodeQL CLI setup
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

CodeQL CLI has subcommands you can execute to verify that you are correctly set
up to create and analyze databases:

- Run ``codeql resolve languages`` to show which languages are
  available for database creation. This will list the languages supported by
  default in your CodeQL CLI package.

> - (Optional) You can download some ":ref"`CodeQL packs <about-codeql-packs>` containing pre-compiled queries you would like to run.

Reviewer: Typo in the link here, probably should be:

```suggestion
- (Optional) You can download some ":ref:`CodeQL packs <about-codeql-packs>`" containing pre-compiled queries you want to run.
```

  To do this, run ``codeql pack download <pack-name> [...pack-name]``, where ``pack-name`` is the name of
  the pack you want to download. The core query packs are a good place to start. They are:

      - ``codeql/cpp-queries``
      - ``codeql/csharp-queries``
      - ``codeql/java-queries``
      - ``codeql/javascript-queries``
      - ``codeql/python-queries``
      - ``codeql/ruby-queries``

>    Alternatively, you can download query packs during the analysis by using the `--download` flag of the `codeql database analyze`

Reviewer: ```suggestion
   Alternatively, you can download query packs during the analysis by using the ``--download`` flag of the ``codeql database analyze``
```

     The queries on the ``lgtm.com`` branch are more likely to be compatible
     with the ``latest`` CLI, so you'll be less likely to have to upgrade
     newly-created databases than if you use the ``main`` branch. Older databases
     may need to be upgraded before you can analyze them.

   - For the most up to date CodeQL queries, check out the ``main`` branch.
     This branch represents the very latest version of CodeQL's analysis. Even
     databases created using the most recent version of the CLI may have to be
     upgraded before you can analyze them. For more information, see
     ":doc:`Upgrading CodeQL databases <upgrading-codeql-databases>`."

   - For the queries used in a particular LGTM Enterprise release, check out the
     branch tagged with the relevant release number. For example, the branch
     tagged ``v1.27.0`` corresponds to LGTM Enterprise 1.27. You must use this
     version if you want to upload data to LGTM Enterprise. For further
     information, see `Preparing CodeQL databases to upload to LGTM
     <https://help.semmle.com/lgtm-enterprise/admin/help/prepare-database-upload.html>`__
     in the LGTM admin help.

4. Extract the zip archive

> --------------------------

Reviewer: I think this is the wrong heading type:

```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~
```
